app/component/itinerary/ItineraryPage.js

```diff
@@ -352,6 +368,79 @@ export default function ItineraryPage(props, context) {
     }
   }
 
+  async function makeScooterQuery(settings) {
+    if (!planQueryNeeded(config, match, PLANTYPE.TRANSIT)) {
+      setScooterState(emptyPlan);
+      setCombinedRentalState(emptyPlan);
+      resetItineraryPageSelection();
+      return;
+    }
+
+    ariaRef.current = 'itinerary-page.loading-itineraries'; // might need to be changed
+    setScooterState({ ...scooterState, loading: LOADSTATE.LOADING }); // check somewhere
+
+    const planParams = getPlanParams(
+      config,
+      match,
+      PLANTYPE.SCOOTERTRANSIT,
+      false, // no relaxed settings
+    );
+
+    const tunedParams = {
+      ...planParams,
+      allowedBikeRentalNetworks: settings.allowedScooterRentalNetworks,
+    };
+    try {
+      const plan = await iterateQuery(tunedParams);
+      setScooterState({ ...scooterState, plan, loading: LOADSTATE.DONE });
+      resetItineraryPageSelection();
+      ariaRef.current = 'itinerary-page.itineraries-loaded'; // fix?
+    } catch (error) {
+      reportError(error);
+      setScooterState({
+        ...scooterState,
+        plan: {},
+        loading: LOADSTATE.UNSET,
+        error,
+      });
+    }
+  }
+
+  async function makeRelaxedScooterQuery() {
+    if (!planQueryNeeded(config, match, PLANTYPE.TRANSIT)) {
```

Comment:
I am not able to see relaxed scooter itineraries. Query is made but standard relaxed itinerary set is shown.

Reply:
The relaxed scooter itineraries are only supposed to show up if no standard relaxed itineraries are found. This is a rather rare situation and I have only been able to test it by breaking the search temporarily by disabling transport modes when the standard relaxed search is made.

Reply:
ok!